docs/pages/heading.js

```diff
@@ -99,64 +335,78 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
       Dark
     </Heading>
   </Box>
-</Box>
+</Flex>
 `}
-      />
-      <Example
-        id="overflowTruncation"
-        name="Example: Overflow & truncation"
-        defaultCode={`
-<Box maxWidth={240} marginTop={-2} marginBottom={-2}>
-  <Box paddingY={2}>
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Overflow & truncation">
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex maxWidth={240} direction="column" gap={8}>
+  <Flex direction="column" gap={2}>
     <Text>breakWord (default):</Text>
-    <Heading size="400" overflow="breakWord" >
-      This is a long and Supercalifragilisticexpialidocious sentence.
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" overflow="breakWord">
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
 
-  <Box paddingY={2}>
+  <Flex direction="column" gap={2}>
     <Text>normal:</Text>
-    <Heading size="400" overflow="normal">
-      This is a long and Supercalifragilisticexpialidocious sentence.
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" overflow="normal">
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
 
-  <Box paddingY={2}>
-    <Text>lineClamp:</Text>
-    <Heading size="400" lineClamp={2}>
-      This is a long and Supercalifragilisticexpialidocious sentence.
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
-</Box>
+  <Flex direction="column" gap={2}>
+  <Text>lineClamp:</Text>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" lineClamp={2}>
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
+</Flex>
 `}
-      />
-      <Example
-        id="align"
-        name="Example: Alignment"
-        defaultCode={`
-<Box>
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Alignment">
```

Comment:
Can we keep text aligned with Text? It's the same section, so we could just opy paste from one another. This one is empty but we could have the same description as in Text.